tests/jest/db/DAO.spec.js

```diff
@@ -423,9 +422,8 @@ test('StudyRequestDAO', async () => {
   // save study request
   let persistedStudyRequest = await StudyRequestDAO.create(transientStudyRequest);
   expect(persistedStudyRequest.id).not.toBeNull();
-  await expect(
-    Joi.validate(persistedStudyRequest, StudyRequest.read),
-  ).resolves.toEqual(persistedStudyRequest);
+  const { value } = StudyRequest.read.validate(persistedStudyRequest);
```

Comment:
please use `await expect(StudyRequest.read.validateAsync(persistedStudyRequest)).resolves.toEqual(persistedStudyRequest);`, as this is closer to the previous implementation